main/sensors/sensors.c

```diff
@@ -0,0 +1,93 @@
+#include "sensors.h"
+#include "adc.h"
+
+static uint64_t count = 0;
```

Comment:
Could you rename this to something more clear? soil_capacity_timer_count or something?

Reply:
Will def break it but alrighty

Reply:
Probably not actually.